decentraland/screens/__snapshots__/MyAccountScreen.test.js.snap

```diff
@@ -0,0 +1,61 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`MyAccountScreen renders the component with account 1`] = `
+<MyAccount
+  creationActions={
+    Array [
+      Object {
+        "action": null,
+        "name": "Account created",
+        "status": "done",
+      },
+      Object {
+        "action": "FUNDING_WITH_ETH",
+        "name": "Funded with ETH",
+        "status": "done",
+      },
+      Object {
+        "action": "FUNDING_WITH_MANA",
+        "name": "Funded with MANA tokens",
+        "status": "missing",
+      },
+      Object {
+        "action": "APPROVING_MARKETPLACE",
+        "name": "Linked to marketplace",
+        "status": "missing",
+      },
+    ]
+  }
+  progress={0}
```

Comment:
Why is progress `0` here, given that this is the example where there's an account and it's funded with ETH?

Reply:
I honestly have no idea, when run on device it show progress properly. Maybe jest handles decimal number poorly?

Reply:
Hmm, maybe. Did we pass in all needed props to the component in this test?